WordPress/Classes/ViewRelated/Domains/Register/PickDomainViewController.swift

```diff
@@ -0,0 +1,99 @@
+import UIKit
+import WordPressAuthenticator
+
+class PickDomainViewController: UIViewController, HidableBottomViewOwner {
```

Comment:
RegisterDomainSuggestionsViewController sounds better. I also added NUXViewController inheritance but couldn't get rid of adding Cancel button because of the reasons I mentioned on the below change set.